src/swagger/paths/classes.yml

```diff
@@ -201,7 +201,7 @@ class-spells-path:
         content:
           application/json:
             schema:
-              $ref: '../schemas/combined.yml#/APIReferenceList'
```

Comment:
fwiw, in a versioned world i would totally look at reworking the actual endpoint implementation for all of these to return a more uniform shape for "spells with a level". but if feels more important to have the docs match the actual response first.